docs/src/components/PlaygroundRenderSettings.tsx

```diff
@@ -1,31 +1,36 @@
-function PlaygroundRenderSettings({ onSubmit }: { onSubmit: () => Promise<void> }) {
+import { useState } from 'react';
+import PlaygroundRenderSettingsInputs from './PlaygroundRenderSettingsInputs';
+
+interface PlaygroundRenderSettingsProps {
+  onSubmit: () => Promise<void>;
+  onChange: (string, Resolution) => void;
+}
+
+function PlaygroundRenderSettings({ onSubmit, onChange }: PlaygroundRenderSettingsProps) {
+  const [inputsSettingsModalOpen, setInputsSettingsModalOpen] = useState(false);
+
   return (
     <div style={{ margin: '10px' }}>
-      <div className="row">
-        <div className="col">Settings:</div>
-        <div className="col">
-          <select>
-            <option value="someOption">Some option</option>
-            <option value="otherOption">Other option</option>
-          </select>
-        </div>
-        <div className="col">
-          <select>
-            <option value="someOption">Some option</option>
-            <option value="otherOption">Other option</option>
-          </select>
-        </div>
-        <div className="col">
-          <select>
-            <option value="someOption">Some option</option>
-            <option value="otherOption">Other option</option>
-          </select>
-        </div>
-        <div className="col">
-          <button className="button button--outline button--primary" onClick={onSubmit}>
-            Submit
-          </button>
-        </div>
+      <div>
+        <button
+          className="button button--outline button--secondary"
+          onClick={() => {
+            setInputsSettingsModalOpen(true);
+          }}
+          style={{ margin: '10px' }}>
+          Inputs settings
+        </button>
+        <PlaygroundRenderSettingsInputs
```

Comment:
If this is something that has to somwhere in the component tree, but it does not matter where then put it e.g. on as the last item in root div?